### PR TITLE
Electron 19 (Frameless Window)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
Added support for frameless window

### CODE REVIEW

**Comment by KiranNiranjan** [comment]
@lneir Please review. Thanks

**Comment by lneir** [comment]
please rebase PR from latest, there are many other PR mixed in here.

File: js/windowMgr.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ function doCreateMainWindow(initialUrl, initialBounds) {
         show: true,
         minWidth: MIN_WIDTH,
         minHeight: MIN_HEIGHT,
+        titleBarStyle: 'hidden',
```
Comment (lneir) [inline js/windowMgr.js]: what does this do on windows?
also looks like we loose capability to move window around?

Comment (lneir) [inline js/windowMgr.js]: also looks like we loose capability to move window around?  maybe change in client fixes this???

Comment (KiranNiranjan) [inline js/windowMgr.js]: This settings will not affect wrapper running in windows.

Comment (KiranNiranjan) [inline js/windowMgr.js]: I have added this ```-webkit-app-region: drag``` class to the header section in client-app which takes care of the draggable window.

Comment (lneir) [inline js/windowMgr.js]: but won't the client code impact?

**Comment by KiranNiranjan** [comment]
Will open a new pull request.

**Pull request closed without merging**